megamek/src/megamek/common/OffBoardHex.java

```diff
@@ -0,0 +1,18 @@
+package megamek.common;
+
+public class OffBoardHex extends Hex {
+
+    OffBoardHex() {
+        super();
+    }
```

Comment:
I _believe_ you can leave the constructor out and it will automatically call the super constructor, but I'd need to double-check that.